client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx

```diff
@@ -246,4 +246,48 @@ describe('Audit Setup > Review & Launch', () => {
       })
     })
   })
+
+  it.only('accepts custom sample size', async () => {
```

Comment:
supposed to be `.only`?

Reply:
@jonahkagan Ah, thanks for the catch. I was doing that to speed up test development, forgot to take it back out again, haha.